docs/DeveloperGuide.md

```diff
@@ -6,32 +6,45 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+1. The following sequence diagram shows how the Bye Command works.
+![](ByeCommand.png)
```

Comment:
Useful diagram to have! But before introducing it, you may want to give an overview of your application in terms of its architecture and how the relations between the different classes.